cfme/tests/cloud/test_providers.py

```diff
@@ -727,42 +871,74 @@ def test_domain_id_validation(request, provider):
 
 @pytest.mark.manual
 @pytest.mark.tier(1)
-def test_upload_azure_image_to_azure():
+def test_sdn_nsg_arrays_refresh_azure():
     """
-    Upload image copied from RHCF3-11271 to azure using powershell
-    similiar the script in the setup section.
+    https://bugzilla.redhat.com/show_bug.cgi?id=1520196
 
     Polarion:
         assignee: anikifor
         casecomponent: Cloud
         caseimportance: medium
-        initialEstimate: 1/2h
-        setup: # Video Demo Script - Upload VHD to Azure
-               # You have to log in via Login-AzureRmAccount
-               Get-AzureRmSubscription -SubscriptionId "9ee63d8e-aee7-4121-861c-
-               d67a5b8d231e" -TenantId "2e64678d-2fa8-40ed-8922-c9b8e2c3f100" |
-               Select-AzureRmSubscription
-               $ResourceGroupName = "CFMEQE-Main"
-               $BlobLocation = "https://cfmeqestore.blob.core.windows.net/"
-               $BlobContainer = "templates"
-               $BlobName = "tmpl-osDisk.vhd"
-               $BlobDestination = $BlobLocation + $BlobContainer + "/" + $BlobName
-               $LocalFilePath = "C:\tmp\"
-               $LocalFileName = "cfme-azure-5.6.0.13-1.x86_64.vhd"
-               $LocalFile = $LocalFilePath + $LocalFileName
-               Add-AzureRmVhd -ResourceGroupName $ResourceGroupName -Destination
-               $BlobDestination -LocalFilePath $LocalFile -NumberOfUploaderThreads 8
+        initialEstimate: 1/6h
+        testSteps:
+            1. Add Network Security group on Azure with coma separated port ranges
+            `1023,1025` rule inbound/outbound ( ATM this feature is not allowed in
+            East US region of Azure - try West/Central)
+            2. Add such Azure Region into CFME
+            3. Refresh provider
+    """
+    pass
+
+
+@pytest.mark.manual
+@pytest.mark.tier(2)
+def test_provider_flavors_azure():
+    """
+    Verify that the vm flavors in Azure are of the correct sizes and that
+    the size display in CFME is accurate.  This test checks a regression
+    of https://bugzilla.redhat.com/show_bug.cgi?id=1357086
```

Comment:
Please use docblock section to list BZ's associated with the manual case (no pytest marker at this time for manual cases)

```
Bugzilla:
    1357086
```